src/components/app/DocumentList/index.js

```diff
@@ -0,0 +1,855 @@
+      });
+  };
+
+  /**
+   * @method filterView
+   * @summary apply filters and re-fetch layout, data. Then rebuild the page
+   */
+  filterView = (locationAreaSearch) => {
+    const {
+      windowType,
+      isIncluded,
+      page,
+      sort,
+      viewId,
+      setListIncludedView,
+      filterView,
+    } = this.props;
+    const { filtersActive } = this.state;
+
+    filterView(windowType, viewId, filtersActive.toIndexedSeq().toArray())
+      .then((response) => {
+        const viewId = response.viewId;
+
+        if (isIncluded) {
+          setListIncludedView({ windowType, viewId });
+        }
+
+        this.mounted &&
+          this.setState(
+            {
+              triggerSpinner: false,
+            },
+            () => {
+              this.getData(viewId, page, sort, locationAreaSearch);
+            }
+          );
+      })
+      .catch(() => {
+        this.setState({ triggerSpinner: false });
+      });
+  };
+
+  /**
+   * @method getData
+   * @summary Loads view/included tab data from REST endpoint
+   */
+  getData = (id, page, sortingQuery, locationAreaSearch) => {
+    const {
+      windowType,
+      selections,
+      updateUri,
+      type,
+      isIncluded,
+      fetchDocument,
+      indicatorState,
+      selectTableItems,
+      updateRawModal,
+      viewId,
+    } = this.props;
+
+    indicatorState('pending');
+
+    if (updateUri) {
+      id && updateUri('viewId', id);
+      page && updateUri('page', page);
+      sortingQuery && updateUri('sort', sortingQuery);
+    }
+
+    return fetchDocument(windowType, id, page, this.pageLength, sortingQuery)
+      .then((response) => {
+        const result = response.result;
+        const resultById = {};
+        const selection = getSelectionDirect(selections, windowType, viewId);
+        const forceSelection =
+          (type === 'includedView' || isIncluded) &&
+          response &&
+          result.length > 0 &&
+          (selection.length === 0 ||
+            !doesSelectionExist({
+              data: result,
+              selected: selection,
+            }));
+
+        result.map((row) => {
+          const parsed = parseToDisplay(row.fieldsByName);
+          resultById[`${row.id}`] = parsed;
+          row.fieldsByName = parsed;
+        });
+
+        const pageColumnInfosByFieldName = response.columnsByFieldName;
+
+        mergeColumnInfosIntoViewRows(
+          pageColumnInfosByFieldName,
+          response.result
+        );
+
+        if (this.mounted) {
+          const newState = {
+            pageColumnInfosByFieldName: pageColumnInfosByFieldName,
+            triggerSpinner: false,
+          };
+
+          if (response.filters) {
+            newState.filtersActive = filtersToMap(response.filters);
+          }
+
+          if (
+            locationAreaSearch ||
+            (newState.filtersActive &&
+              newState.filtersActive.has(`location-area-search`))
```

Comment:
use single quotes pls. Note that all these even if you find them non-important are important because overall they improve our code quality and also we will have greater codacy mark.

Reply:
That doesn't mean codacy is any good, because it's against what everyone else uses on the market. Backticks are what everyone is moving to now.